chatAppTemplate_2/chatApp.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html >
+  <head>
+    <meta charset="UTF-8">
+    <title>Superb Owl</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    	<div class="ui">
+		<div class="left-menu">
+				<form action="#" class="search">
+					<input placeholder="search..." type="search" name="" id="">
+					<input type="submit" value="&#xf002;">
+				</form>
+				<menu class="list-friends">
+					<li>
+						<img width="50" height="50" src="http://cs625730.vk.me/v625730358/1126a/qEjM1AnybRA.jpg">
+						<div class="info">
+							<div class="user">Online Test</div>
+							<div class="status on"> online</div>
+						</div>
+					</li>
+					<li>
+						<img width="50" height="50" src="http://lorempixel.com/50/50/people/8">
+						<div class="info">
+							<div class="user">Offline Test</div>
+							<div class="status off">offline</div>
+						</div>
+					</li>
+				</menu>
+		</div>
+		<div class="chat">
+			<div class="top">
+				<div class="avatar">
+					<img width="50" height="50" src="http://cdn.backyardchickens.com/6/6e/6eecbcc8_Great-Horned-Owl-flat-best-.jpeg">
+				</div>
+				<div class="info">
+					<div class="name">Superb Owl Chat</div>
+					<div class="count"></div>
+				</div>
+			</div>
+			<ul class="messages">
+
+			</ul>
+			<div class="write-form">
+				<textarea placeholder="Type your message" name="e" id="text" rows="2"></textarea>
+				<span class="send">Send</span>
+			</div>
+		</div>
+	</div>
+    <script src='http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
```

Comment:
Does HTML support comments? If so - please comment what these two scripts are and what they are doing. 

Also lets host these scripts ourselves in our repo instead of reaching out and grabbing them.